client/src/models/models/model_0/input.py

```
@@ -0,0 +1,26 @@
+from __future__ import annotations
+from pydantic import BaseModel
```

Flake8 (7.2.0)

[error] 'pydantic.BaseModel' imported but unused

(F401)